src/WPF/ArcGISRuntime.WPF.Viewer/MainWindow.xaml.cs

```diff
@@ -1,100 +1,87 @@
 //
-// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 // language governing permissions and limitations under the License.
 
 using ArcGISRuntime.Samples.Managers;
-using ArcGISRuntime.Samples.Models;
+using ArcGISRuntime.Samples.Shared.Models;
+using Esri.ArcGISRuntime.Security;
 using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using System.Windows;
 using System.Windows.Controls;
-using System.Windows.Media;
-using System.Windows.Media.Imaging;
 
 namespace ArcGISRuntime.Samples.Desktop
 {
     public partial class MainWindow
     {
-        private bool _isTakingThumbnail;
-        private int _previousHeight;
-        private int _previousWidth;
-
+        private bool _waitFlag;
         public MainWindow()
         {
             InitializeComponent();
             Initialize();
         }
 
-        private async void Initialize()
+        private void Initialize()
         {
             try
             {
-                await SampleManager.Current.InitializeAsync(ApplicationManager.Current.SelectedLanguage);
-
-                // Set featured data context
-                featured.DataContext = SampleManager.Current.GetFeaturedSamples();
-                featured.SelectedIndex = 0;
+                SampleManager.Current.Initialize();
 
                 // Set category data context
-                categories.DataContext = SampleManager.Current.GetSamplesInTreeViewCategories();
+                Categories.DataContext = WPF.Viewer.Helpers.ToTreeViewItem(SampleManager.Current.FullTree);
+
+                // Select the first item
+                ((List<TreeViewItem>)Categories.DataContext).First().IsSelected = true;
             }
             catch (Exception ex)
             {
                 MessageBox.Show(ex.ToString(), "Initialization exception occurred.");
             }
         }
 
-        private void featured_SelectionChanged(object sender, SelectionChangedEventArgs e)
-        {
-            if (e.AddedItems.Count > 0)
-            {
-                var featuredItem = e.AddedItems[0] as FeaturedModel;
-                SelectSample(featuredItem.Sample);
-            }
-        }
-
         private void categoriesList_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
             if (e.AddedItems.Count > 0)
             {
-                var sample = e.AddedItems[0] as SampleModel;
+                var sample = e.AddedItems[0] as SampleInfo;
                 SelectSample(sample);
```

Comment:
This is a non-awaited async-void call. Are you OK with the rest of this method executing before sample is done being selected?